examples/waspello/src/cards/Drappable.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react'
+import { Droppable as RBDDroppable } from 'react-beautiful-dnd'
+
+
+export const Droppable = ({ children, ...props }) => {
```

Comment:
Could you auto format this file? 

It's missing a new line at the end and the indentation feels off.